### ISSUE TITLE
General error: 5 database is locked

### ISSUE THREAD

**Issue by badnetmask** [kanboard/kanboard#5248]
### Checklist

- [X] I verified that Kanboard is correctly installed
- [X] I verified that the problem does not come from a plugin
- [X] I verified that the problem is not already reported
- [X] I understand that Kanboard is in maintenance mode. It doesn't mean it's abandoned, but there is no significant feature development

### Actual behaviour

Application error while using Docker image:
```
[error] 18#18: *765 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Uncaught PicoDb\SQLException: SQL Error[HY000]: SQLSTATE[HY000]: General error: 5 database is locked in /var/www/app/libs/picodb/lib/PicoDb/StatementHandler.php:353
```

### Expected behaviour

In theory, this was supposed to have been fixed by #5135, as reported on #4913.

### Steps to reproduce

- Git clone this repo.
- `docker-compose up`.
- Create cards
- Update cards and/or move them around

### Screenshots

N/A

### Logs

```
kanboard_1  | 2023/04/19 15:16:07 [error] 18#18: *765 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Uncaught PicoDb\SQLException: SQL Error[HY000]: SQLSTATE[HY000]: General error: 5 database is locked in /var/www/app/libs/picodb/lib/PicoDb/StatementHandler.php:353
kanboard_1  | Stack trace:
kanboard_1  | #0 /var/www/app/libs/picodb/lib/PicoDb/StatementHandler.php(245): PicoDb\StatementHandler->handleSqlError()
kanboard_1  | #1 /var/www/app/libs/picodb/lib/PicoDb/Database.php(257): PicoDb\StatementHandler->execute()
kanboard_1  | #2 /var/www/app/libs/picodb/lib/PicoDb/Table.php(226): PicoDb\Database->execute()
kanboard_1  | #3 /var/www/app/app/Core/Session/SessionHandler.php(68): PicoDb\Table->update()
kanboard_1  | #4 [internal function]: Kanboard\Core\Session\SessionHandler->write()
kanboard_1  | #5 [internal function]: session_write_close()
kanboard_1  | #6 {main}
kanboard_1  |   thrown in /var/www/app/libs/picodb/lib/PicoDb/StatementHandler.php on line 353" while reading response header from upstream, client: 10.0.0.10, server: localhost, request: "GET /?controller=BoardAjaxController&action=check&project_id=1&timestamp=1681916357&csrf_token=<redacted>&_=<redacted> HTTP/2.0", upstream: "fastcgi://unix:/var/run/php-fpm.sock:", host: "10.0.0.1:443"
```

### Configuration

```
Application version: v1.2.28
PHP version: 8.1.17
PHP SAPI: fpm-fcgi
HTTP Client: cURL
OS version: Linux 6.1.19-v8+
Database driver: sqlite
Database version: 3.40.1
```

**Comment by harry75369** [comment]
Same for version `v1.2.29`

**Comment by YorVeX** [comment]
Has been happening forever and is still in v1.2.30

**Comment by tomeli5n** [comment]
> Has been happening forever and is still in v1.2.30 - the direct next attempt works, so if it's hard to find or fix, could we at least have a mechanism that just retries a single time 500 ms later if this error occurs? I'd assume that fixes 99% of cases with this error.

Hi yorvex, what do you mean by "the direct next attemp"?
Do you know a way , at least, from hide the error from the frontend?
Disable the debug mode does not work, the error messages are still displayed 
![image](https://github.com/kanboard/kanboard/assets/7417323/40797b42-adbf-4c33-b0ac-1e4ffbaf6f03)

**Comment by YorVeX** [comment]
I get this error e.g. when I move a task from one column to another. Then afterwards I just do it again and the second time it works, although I do the exact same thing as before. Which tells me it's some kind of a race condition that occurs with a random chance.
